packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderFirst/en.ftl

```diff
@@ -1,7 +1,7 @@
-verificationReminderFirst-subject = Reminder: Finish creating your account
-verificationReminderFirst-title = Welcome to the { -brand-firefox } family
-verificationReminderFirst-description = A few days ago you created a { -product-firefox-account }, but never confirmed it.
-verificationReminderFirst-sub-description = Confirm now and get technology that fights for and protects your privacy, arms you with practical knowledge, and the respect you deserve.
-confirm-email = Confirm email
-confirm-email-plaintext = { confirm-email }:
-verificationReminderFirst-action = Confirm email
+verificationReminderFirst-subject-2 = Remember to confirm your account
+verificationReminderFirst-title-2 = Welcome to { -brand-firefox }!
+verificationReminderFirst-description-2 = A few days ago you created a { -product-firefox-account }, but never confirmed it. Please confirm your account within the next 15 days or it will be automatically deleted.
+verificationReminderFirst-sub-description-2 = Don’t miss out on tech that puts you and your privacy first.
+confirm-email-2 = Confirm account:
```

Comment:
confirm-email-2 contains a colon, but looking at the mock-up this appears to be the button text. Should the colon be removed? 

Or is verificationReminderFirst-action-2 the button text? If so, I'm not quite sure about the context where this appears.

Reply:
It's a bit of an odd pattern to me, too -- it looks like it's just a variable that gets assigned into `confirm-email-plaintext-2` below. There seems to be a pattern (also visible in the `verificationReminderSecond` email, for example) where in the standard version of the email, you get button with the CTA (sans colon), and then in the plaintext version, you get the same CTA, but as  text followed by a colon, followed by a link. Like this (seen in storybook):
<img width="312" alt="Screen Shot 2022-08-10 at 2 19 06 PM" src="https://user-images.githubusercontent.com/11150372/184022265-5f2faf73-6b0f-47b6-9e56-8a0b8db5c1b7.png">
<img width="469" alt="Screen Shot 2022-08-10 at 2 19 08 PM" src="https://user-images.githubusercontent.com/11150372/184022267-f927b0af-e6ea-46ef-ace4-bea106381be9.png">

Reply:
Interesting, but if that's the case then wouldn't having `Confirm email:` being called into `{ confirm-email-2 }:` then result in `Confirm email::` ?

Reply:
🤦 Yeah it totally would. I'm so sorry I missed that, updating now

Reply:
Thanks! Looks good now 👍